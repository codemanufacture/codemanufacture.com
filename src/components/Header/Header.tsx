import * as React from 'react'
import styled from 'styled-components'
import Logo from '../Logo'
import MainMenu from '../MainMenu'

interface HeaderProps {
  siteTitle: string
}

const StyledHeaderWrapper = styled.div`
  display: flex;
  justify-content: space-between;
  height: 70px;
  margin: 0 auto;
  padding: 0 30px;
`

const Header: React.SFC<HeaderProps> = ({ siteTitle }) => (
  <StyledHeaderWrapper>
    <Logo alt={siteTitle} width={260} height={40} />
    <MainMenu />
  </StyledHeaderWrapper>
)

export default Header
