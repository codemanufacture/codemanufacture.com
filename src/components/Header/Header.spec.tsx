import * as React from 'react'
import renderer from 'react-test-renderer'
import Header from './Header'

describe('Header component', () =>
  it('renders correctly', () => {
    const tree = renderer
      .create(<Header siteTitle="Codemanufacture" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  }))
