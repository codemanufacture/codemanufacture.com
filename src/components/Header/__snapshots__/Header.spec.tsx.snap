// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders correctly 1`] = `
<div
  className="sc-ifAKCX cNOviL"
>
  <a
    aria-current="page"
    className="sc-bdVaJa dBkIVO"
    href="/"
    onClick={[Function]}
    onMouseEnter={[Function]}
    style={Object {}}
    title="Back to homepage"
  >
    <img
      alt="Codemanufacture"
      height={40}
      src={
        Object {
          "0": "t",
          "1": "e",
          "10": "s",
          "11": "t",
          "12": "u",
          "13": "b",
          "2": "s",
          "3": "t",
          "4": "-",
          "5": "f",
          "6": "i",
          "7": "l",
          "8": "e",
          "9": "-",
          "default": "test-file-stub",
        }
      }
      width={260}
    />
  </a>
  <menu
    className="sc-bxivhb ktNeoN"
  >
    <ul
      className="sc-htpNat hyuzrE"
      onClick={[Function]}
    >
      <li>
        <a
          href="/homepage#home"
          onClick={[Function]}
          onMouseEnter={[Function]}
          title="Home"
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="/homepage#services"
          onClick={[Function]}
          onMouseEnter={[Function]}
          title="Services"
        >
          Services
        </a>
      </li>
    </ul>
    <a
      className="sc-bwzfXH bvpJTi"
      href="#trigger-mobile-menu"
      onClick={[Function]}
    >
      Menu
    </a>
  </menu>
</div>
`;
